spec.md

```diff
@@ -22,11 +21,9 @@
   - [Broker Errors](#broker-errors)
   - [Orphans](#orphans)
 
-<div id="changes"/> 
-## Changes ##
+## <div id="changes"/> Changes
```

Comment:
@duglin This pr attempts to fix the styling for all the headers in the current spec.md doc. Thanks, pls let me know if you want any further clarification.